### PR TITLE
Add an explanation of USDC for the buy modal.

### PR DESCRIPTION
**Pull request by alantoa**
# Why

# How

# Test Plan

### CODE REVIEW

**Comment by hirbod** [comment]
Waiting for Vercel preview build to test

**Comment by hirbod** [comment]
@alantoa lets add the buy/sell toggle change to this PR as well.

File: packages/app/components/creator-token/buy-creator-token.tsx [hunk 1]
```diff
@@ -281,7 +301,7 @@ export const BuyCreatorToken = () => {
                 <Skeleton width={40} height={16} />
               ) : (
                 <Text tw="font-semibold text-gray-700 dark:text-gray-200">
-                  {tokenBalance.data?.toString()}
+                  {tokenBalance.data?.toString() ?? "N/A"}
```
Comment (hirbod) [inline packages/app/components/creator-token/buy-creator-token.tsx:277]: Added N/A fallback when no tokenbalance is known

Comment (alantoa) [reply packages/app/components/creator-token/buy-creator-token.tsx:277]: yeah, makes sense!

**Comment by hirbod** [comment]
@alantoa has a conflict now

**Pull request merged**